packages/react-dom/src/shared/ReactControlledValuePropTypes.js

```diff
@@ -15,6 +15,13 @@ const ReactControlledValuePropTypes = {
   checkPropTypes: null,
 };
 
+const propsAreConfigured = hasReadOnlyValue[props.type] ||
```

Comment:
This should be in the __DEV__ block, otherwise it won't be dead code removed for prod.